Updated Monitoring and Logging info (#51761)
kubernetes/website: content/en/docs/tasks/debug/_index.md

<!-- overview -->

> Sometimes things go wrong. This guide is aimed at helping the users with getting the relevant information and making them right. It has

Reviewer: ```suggestion
Sometimes things go wrong. This guide helps you gather the relevant information and resolve issues. It has four sections:
```

> four sections:

Reviewer: ```suggestion

```

* [Debugging your application](/docs/tasks/debug/debug-application/) - Useful

>   for users who are deploying code into Kubernetes and wondering why it is not working.

Reviewer: ```suggestion
  for users deploying code into Kubernetes and troubleshooting why it is not working.
```

* [Debugging your cluster](/docs/tasks/debug/debug-cluster/) - Useful

>   for cluster administrators and people whose Kubernetes cluster is unhappy.

Reviewer: ```suggestion
  for cluster administrators and operators troubleshooting issues with the Kubernetes cluster itself.
```

* [Logging in Kubernates](/docs/tasks/debug/logging/) - Useful
  for cluster administrators and people for enabling logging mechanism in Kubernates.
* [Monitoring in Kubernates](/docs/tasks/debug/monitoring/) - Useful

>   for cluster administrators and people to enable monitoring in Kubernetes cluster.

Reviewer: ```suggestion
  for cluster administrators who want to enable monitoring in a Kubernetes cluster.
```

* [Debugging your cluster](/docs/tasks/debug/debug-cluster/) - Useful
  for cluster administrators and people whose Kubernetes cluster is unhappy.

Logging in Kubernetes. This guide will help in information like how to collect, access, and analyze logs in Kubernetes using built-in tools and popular logging stacks:

* [Logging Architecture](/docs/concepts/cluster-administration/logging/)
* [System Logs](/docs/concepts/cluster-administration/system-logs/)
* [A Practical Guide to Kubernetes Logging](https://www.cncf.io/blog/2020/10/05/a-practical-guide-to-kubernetes-logging)

Monitoring in Kubernetes. This guide will give information about system metrics and Tracing:

* [Metrics For Kubernetes System Components](/docs/concepts/cluster-administration/system-metrics/)

> * [Traces For Kubernetes System Components](docs/concepts/cluster-administration/system-traces/)

Reviewer: ```suggestion
* [Traces For Kubernetes System Components](/docs/concepts/cluster-administration/system-traces/)
```
Broken Link

Author: Updated the broken link and structured the logging and monitoring sections
